tools/downloader/pytorch_to_onnx.py

```diff
@@ -9,19 +10,28 @@
 import torch.onnx
 
 
+def is_sequence(element):
+    return isinstance(element, (list, tuple))
+
+
 def positive_int_arg(values):
     """Check positive integer type for input argument"""
```

Comment:
```suggestion
def shapes_arg(values):
    """Checks that the argument represents a tensor shape or a sequence of tensor shapes"""
```

This function name was already wrong before, but now it's just grotesquely wrong. Might as well fix it. :-)